assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/outputs.tf

```diff
@@ -62,3 +62,7 @@ output "kubelet_values" {
 output "calico_values" {
   value = module.bootkube.calico_values
 }
+
+output "webhook-admission_values" {
```

Comment:
I'm still not a fan of `webhook-admission` name. I think it is too specific. Maybe let's call it `lokomotive` chart for now, as an umbrella chart for Lokomotive webhooks, Lokomotive operator etc?